frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -160,8 +164,19 @@ const handleSubmit = async () => {
         errorCode,
         role
     );
+
     setCurrentTabState(TabKey.UserAccess);
-    router.push('/dashboard');
+
+    if (newUserAccessIds.length > 0) {
+        router.push({
+            path: `/${routeItems.dashboard.name}`,
+            query: {
+                newUserAccessIds: newUserAccessIds.join(','),
```

Comment:
we need to be really clear about the query parameter type, they will all be string, so for example '1' for single, or '1,2,3' for multiple